starter-code/functions-and-arrays.js

```diff
@@ -25,7 +29,11 @@ console.log(longest);
 
 // Calculating a Sum
 function sumArray (array) {
-
+  var sum = 0;
+  array.forEach(function(elmt){
```

Comment:
it's ok to use generic names for generic functions but try to make variable name readable